openfisca_core/simulations.py

```diff
@@ -11,11 +11,11 @@
 
 
 class Simulation(object):
-    compact_legislation_by_instant_cache = None
+    legislation_at_instant_cache = None
```

Comment:
This breaking change must be documented in the changelog.

Maybe we should also make this attribute private ?